interface/billing/billing_process.php

```diff
@@ -23,7 +23,8 @@
 
 use OpenEMR\Billing\BillingUtilities;
 use OpenEMR\Billing\Hcfa1500;
-use OpenEMR\Billing\X12_5010_837P;
+use OpenEMR\Billing\X125010837I;
+use OpenEMR\Billing\X125010837P;
```

Comment:
nice, they are starting to look like UUIDs (in the future, I predict all communication will be via UUIDs)
(bad joke)

Reply:
:), they are something only a mother could love, but with another version in draft coming along, asc x12 7030 figured we could live with psr12 5010, somehow

Reply:
should have mentioned that the `PSR4` move of Jerry's 837 is included